Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -225,10 +247,10 @@ - (void)testReportWithAppCrashingAndRetryOnlyFailedTestsSet {
     self.config.errorRetriesCount = @1;
     self.config.failureTolerance = @1;
     self.config.onlyRetryFailed = YES;
-
+    
```

Comment:
Please don't leave whitespace. You can make Xcode remove it automatically in settings:

<img width="912" alt="image" src="https://user-images.githubusercontent.com/4950/49888117-fb9bba00-fdf2-11e8-9c69-c253bc75d347.png">